### DISCUSSION TITLE
Possible Composition feature for launcher configurations

### DISCUSSION THREAD

**Discussion by akazemipour** [facebookresearch/hydra#2700] [Q&A] [answered]
Hi lovely Hydra developers,
I'm desperately trying to find a way to alternate between different Launcher settings without directly changing them via command line or in the main config.yaml file.
I mean I'd like to have a group configs for the launcher to just specify the desired one at the runtime. How is it possible in your opinion?

**Comment by odelalleau** [comment, marked as answer]
Discussion is moved to issue #2701
